ding/bonus/ppof.py

```diff
@@ -27,6 +28,17 @@ class TrainingReturn:
     wandb_url: str
 
 
+@dataclass
+class EvalReturn:
```

Comment:
can we use the same EvalReturn for different policies

Reply:
fixed